packages/flutter/test/material/tooltip_test.dart

```diff
@@ -1627,6 +1930,55 @@ void main() {
     expect(tip.size.height, equals(64.0));
   });
 
+  testWidgetsWithLeakTracking('Material3 - Tooltip text scales with textScaleFactor', (WidgetTester tester) async {
+    Widget buildApp(String text, { required double textScaleFactor }) {
+      return MediaQuery(
+        data: MediaQueryData(textScaleFactor: textScaleFactor),
+        child: Directionality(
```

Comment:
nit: just a few more😄! Can we also do same changes as above:) ?

Reply:
Good catch, I forgot the theme class too.